amo_storage.py

```diff
@@ -12,24 +12,25 @@
 
 def create_app(**config_overrides):
     app = Flask(__name__)
+    INIConfig(app)
     CORS(app, supports_credentials=True)
 
-    app.config.from_object(AmoStorageConfig)
-
+    #app.config.from_object(AmoStorageConfig)
+    app.config.from_inifile('config.ini', objectify=True)
```

Comment:
Need some last touch: take a command-line option to specify a configuration file location. Of course, there should be a hard-coded default location.

Reply:
I updated the code with your comment reflected.